test/ttmlir/Dialect/TTIR/reshape/reshape_test.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: ttmlir-opt --ttir-reshape-fold %s| FileCheck %s
+#any_device_tile = #tt.operand_constraint<dram|l1|tile|none|interleaved|single_bank|height_sharded|width_sharded|block_sharded|any_layout|any_device_tile>
+module @jit_ravel attributes {mhlo.num_partitions = 1 : i32, mhlo.num_replicas = 1 : i32} {
```

Comment:
Do you need these attributes? Is jit_ravel a good name?

Reply:
You are correct, fixed now.